### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by huynhsontung** [huynhsontung/Indirect#80]
- Make app navigable with just keyboard.
- Controls should be well labeled to work with screen reader.

**Comment by huynhsontung** [comment]
Added UI automation and improved navigation on the keyboard in v.1.4.2. To be continuously improved in the future.
